explainer.md

```diff
@@ -35,22 +35,39 @@ The use cases include:
 * Background removal
 * Voice processing
 
+## Breakout Box Properties
+
+The Breakout Box conceptually splits the MediaStreamTrack into two components:
+* The TrackProcessor, which consumes a MediaStreamTrack's source and generates a stream of
+media frames (VideoFrame or AudioFrame)
+* The TrackGenerator, which consumes a stream of media frames and exposes a MediaStreamTrack
+interface, so that it can be used anywhere a MediaStreamTrack is currently attched.
+
+There is no connection between the stream generator and stream consumer that doesn't pass
+through Javascript; one can freely interconnect the streams in whatever fashion one
+sees fit.
 
 ## Code Examples
 
+Example of processing media before transmitting to a PeerConnection in Phase Two:
+<pre>
+originalTrack = navigator.getUserMedia(...)
+myProcessingElement = new TransformStream(....)
```

Comment:
...?

Reply:
{init() {... }, {transform(frame, controller) { ... }} seemed like a bit excessive to write down, given that this is defined by the TransformStream spec.